src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const discountSmall = 20;
+  const discountLarge = 50;
+  const fullPrice = days * 40;
+
+  if (days >= 7) {
+    result = fullPrice - discountLarge;
+
+    return result;
+  } else if (days >= 3) {
+    result = fullPrice - discountSmall;
+
+    return result;
+  } else {
+    result = fullPrice;
+
+    return result;
```

Comment:
```suggestion
  if (days >= 7) {
    return (fullPrice - discountLarge);
  } else if (days >= 3) {
    return (fullPrice - discountSmall);
  }
  return fullPrice;
```

wrap expressions in brackets.
As you may see, without unnecessary variable "result" the code becomes more cleaner

Also, you don't need the last "else" because if the first two checks won't pass, the function will return the last result by default